PaddleCV/PaddleDetection/ppdet/utils/eval_utils.py

```diff
@@ -36,7 +36,6 @@ def parse_fetches(fetches, prog=None, extra_keys=None):
     for k, v in fetches.items():
         if hasattr(v, 'name'):
             keys.append(k)
-            v.persistable = True
```

Comment:
If remove `v.persistable = True`, maybe also need to set  BuildStrage.inplace=False.

Reply:
Remain this code for now. Delete it after confirm the models precision when BuildStrage.inplace is set to False.